packages/flame_3d/example/lib/scenarios/boxes_setup.dart

```diff
@@ -0,0 +1,79 @@
+import 'dart:math';
+
+import 'package:example/components/crate.dart';
+import 'package:example/components/rendered_point_light.dart';
+import 'package:example/components/rotating_light.dart';
+import 'package:example/example_game_3d.dart';
+import 'package:example/scenarios/game_setup.dart';
+import 'package:flame/extensions.dart';
+import 'package:flame/palette.dart';
+import 'package:flame_3d/components.dart';
+import 'package:flame_3d/resources.dart';
+
+class BoxesScenario implements GameScenario {
+  static final BoxesScenario instance = BoxesScenario._();
```

Comment:
How come it needs to be a singleton?

Reply:
it is twice right now, I think it is more of an access pattern than an actual performance optimization, but I can remove, no problem